extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,259 @@
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "https://ns.adobe.com/experience/campaign/message/reason": {
+          "title": "Reason",
+          "type": "integer",
+          "meta:enum": {
+            "Undefined": "Not defined",
+            "UnknownUser": "User unknown",
+            "InvalidDomain": "Invalid domain",
+            "Unreachable": "Unreachable",
+            "Disabled": "Account disabled",
+            "MailboxFull": "Mailbox full",
+            "NotConnected": "Not connected",
+            "Refused": "Refused",
+            "ErrorIgnored": "Error ignored",
+            "AddressUndefined": "Address not specified",
+            "Blacklisted": "Blacklisted address",
+            "Quarantine": "Quarantined address",
+            "Duplicate": "Duplicate",
+            "TypologyRule": "Excluded by a SQL rule",
+            "BusinessRanking": "Excluded after arbitration",
+            "Cancelled": "Delivery canceled",
+            "Quality": "Bad-quality address",
+            "Unchecked": "Unqualified address",
+            "OfferMissing": "Not eligible for the offers",
+            "OverDeliveryLimit": "Target limited in size",
+            "Expired": "Expired",
+            "TooLong": "Text too long",
+            "Untranslatable": "Character not supported by encoding",
+            "ControlGroup": "Control address"
+          },
+          "description":
+            "The reason why the message could/would not be delivered.",
+          "default": "Undefined"
+        },
+        "https://ns.adobe.com/experience/campaign/message/reasonMessage": {
+          "title": "Reason Message",
+          "type": "string",
+          "description":
+            "Reason of message delivery failure when the status is `bounce` or `nonDeliverable`."
+        },
+        "https://ns.adobe.com/experience/campaign/message/ip": {
+          "title": "IP Address",
+          "oneOf": [
+            {
+              "format": "ipv4"
+            },
+            {
+              "format": "ipv6"
+            }
+          ],
+          "description": "Outbound IP address used to deliver the message."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery/id": {
+          "title": "DeliveryID",
+          "type": "integer",
+          "description": "The campaign activity originating this message."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery/channel": {
+          "title": "Communication Channel",
+          "type": "string",
+          "meta:enum": {
+            "https://ns.adobe.com/xdm/channels/email": "Email",
+            "https://ns.adobe.com/xdm/channels/sms": "Mobile (SMS)",
```

Comment:
enum values should *not* have spaces (or parens)

Reply:
Aren't these enum Labels?

Reply:
These are labels.

Reply:
So then what are the actual values that can be used?

Reply:
As the `type: string` says: any string is valid, but only the values listed in `meta:enum` (the URLs like https://ns.adobe.com/xdm/channels/sms) are understood by Campaign.

Reply:
I see - I was reading the enum backwards!

Reply:
👍